appveyor.yml

```diff
@@ -1,8 +1,8 @@
 version: '{build}'
 skip_tags: true
-image: Visual Studio 2017
+image: Visual Studio 2019
 configuration: Release
-test: off
+test: on
```

Comment:
Tests are executed via Build.ps1, so we don't use the AppVeyor integrated testing, here.

Reply:
ok